packages/design-tokens/docs/pages/how-to-switch-themes.stories.mdx

```diff
@@ -1,6 +1,5 @@
 import { Meta } from '@storybook/addon-docs'
```

Comment:
Should these docs be moved into the `components` package now that you've moved the ThemeProvider over?